datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -100,6 +100,40 @@ select * from foo where ts != '2000-02-01T00:00:00';
 statement ok
 drop table foo;
 
+
+##########
```

Comment:
Could you also possibly add an invalid timezone test like

```sql
❯ select arrow_cast('2021-01-02T03:04:00', 'Timestamp(Nanosecond, Some("Foo"))');
Optimizer rule 'simplify_expressions' failed
caused by
Arrow error: Parser error: Invalid timezone "Foo": 'Foo' is not a valid timezone
```

Reply:
Added in both the integration sqllogictests, and in the datafusion expr tests.